### PR TITLE
Removed surplus back command that was causing AppShare workload to fail

### PR DESCRIPTION
**Pull request by scojac01**
The appshare workload was failing as back was being called twice and therefore the share button was not present to send the test image through Skype correctly.

### CODE REVIEW

File: wlauto/workloads/appshare/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java [hunk 1]
```diff
@@ -78,9 +76,9 @@ private void setupGooglePhotos() throws Exception {
 
     private void sendToGmail(String recipient) throws Exception {
         String gID = gmail.packageID;
-
+        
         shareUsingApp("Gmail", "gmail");
-
+
```
Comment (marcbonnici) [inline wlauto/workloads/appshare/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java]: Can these extra spaces be removed?

File: wlauto/workloads/appshare/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java [hunk 2]
```diff
@@ -102,15 +100,15 @@ private void sendToGmail(String recipient) throws Exception {
 
     private void logIntoSkype(String loginName, String loginPass)  throws Exception {
         shareUsingApp("Skype", "skype_setup");
-
+
```
Comment (marcbonnici) [inline wlauto/workloads/appshare/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java]: Spaces

File: wlauto/workloads/appshare/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java [hunk 3]
```diff
@@ -102,15 +100,15 @@ private void sendToGmail(String recipient) throws Exception {
...
+        
         skype.handleLoginScreen(loginName, loginPass);
 
         sleep(10); // Pause while the app settles before returning
     }
 
     private void sendToSkype(String contactName) throws Exception {
         shareUsingApp("Skype", "skype");
-
+
```
Comment (marcbonnici) [inline wlauto/workloads/appshare/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java]: Spaces

File: wlauto/workloads/appshare/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java (hunk 1 above)
Comment (scojac01) [inline wlauto/workloads/appshare/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java]: Are they actually extra spaces? I think it's just showing that I deleted a blank line and then re-added it to keep the original format?

Comment (marcbonnici) [inline wlauto/workloads/appshare/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java]: Yea they are, I've checked the file in my text editor and can also see the spaces there. They've most likely been added automatically by your editor when you re-added the line.

**Comment by scojac01** [comment]
Spaces now removed.

**Comment by marcbonnici** [comment]
Thanks, can you also merge these 2 commits together?

**Comment by marcbonnici** [comment]
Actually that's fine as 2 commits although could you change the commit message of the 2nd commit to indicate it is just tidying?

**Comment by scojac01** [comment]
Updated commit message to reflect that this was tidying up.

Thanks.

**Comment by scojac01** [comment]
Updated commit message as requested.

**Pull request merged**